src/tests/CommentList.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import CommentList from "../components/CommentList";
+
+const comments = [
```

Comment:
Should this block be moved into a `validProps` object inside the test body?

Reply:
I think this depends, it's ok to have them here (can be reused if more tests are written), but I'm not sure onthe best practice tbh.

Reply:
I think this depends, it's ok to have them here (can be reused if more tests are written), but I'm not sure onthe best practice tbh.

Reply:
Not of major consequence, just a suggestion. Either works